packages/grid/_modules_/grid/components/icons/index.tsx

```diff
@@ -38,6 +38,21 @@ export const ColumnIcon = createSvgIcon(
 
 export const SeparatorIcon = createSvgIcon(<path d="M11 19V5h2v14z" />, 'Separator');
 
+export const ViewHeadlineIcon = createSvgIcon(
```

Comment:
I think it would be better to rename those icons in density small, medium... As atm we don't understand how they are being used

Reply:
The convention is to use the name given in https://material.io/icons, as if the icon was imported from `@material-ui/icons`.

Reply:
I figured it was like what @oliviertassinari that's why I named them like that. Also if they are named like this we can reuse then for other use cases later on if needed. If it is ok I'll resolve this conversation and keep their names as it is.